cmd/filegen/main.go

```diff
@@ -203,6 +198,7 @@ func generateFiles(ctx *cli.Context) error {
 		totalAddressesWithBalances = ctx.GlobalInt(numAddressesWithBalances.Name)
 	} else {
 		totalAddressesWithBalances = numOfShards*(numOfNodesPerShard+numOfObserversPerShard) + numOfMetachainNodes + numOfMetachainObservers
+		totalAddressesWithBalances = numOfShards*(numOfNodesPerShard+numOfObserversPerShard) + numOfMetachainNodes + numOfMetachainObservers
```

Comment:
why is this doubled?

Reply:
I think I pressed cmd+D by mistake and did not see it. Fixed